client/lib/impure-lodash/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import {
```

Comment:
I think the hack here is ok for now, but maybe we should move this to the test helper folder?